litellm/types/llms/openai.py

```diff
@@ -454,7 +454,8 @@ class ChatCompletionUsageBlock(TypedDict):
     prompt_tokens: int
     completion_tokens: int
     total_tokens: int
-
+    cache_creation_input_tokens: Optional[int]
```

Comment:
it might be better to inherit from this class in types/utils.py, instead of modifying the openai block @Penagwin

Reply:
I removed the changes to the openai types